exercises/strain/StrainExample.h

```diff
@@ -0,0 +1,10 @@
+#import <Foundation/Foundation.h>
+
+@interface NSArray (StrainExample)
+
+-(instancetype _Nonnull)keep:(BOOL)compare where:(BOOL (^_Nonnull)(id _Nonnull arrayElement))completionHandler;
```

Comment:
If you add `NS_ASSUME_NONNULL_BEGIN` and `NS_ASSUME_NONNULL_END`, then only `nullable` parameters will need to be indicated.

Here's an example: https://github.com/exercism/objective-c/blob/master/exercises/series/SeriesExample.h